requirements.txt

```diff
@@ -1,4 +1,4 @@
-black
+black==19.3b0
```

Comment:
Now that I think about it, part of me is wondering if this is a good idea.  By fixing the version we do avoid surprise errors, but we are also potentially losing out on formatting updates.  Perhaps the better solution would be to destroy/recreate our virtual environments each time we create a feature branch to insure that we have up to date dependencies? I'm not sure.

Reply:
Good point for missing out formatting updates. The problem here is it's not under our control and makes our CI inconsistent. 

How about updating this version periodically when we want?

Reply:
yeah, the CI inconsistency is a good point.  I think that alone is a good enough reason to lock the version, and periodically updating it would insure that we can receive updates in a more controlled manner.